### ISSUE TITLE
Arrow extends over rounded corner

### ISSUE THREAD

**Issue by Eckelf** [aheze/Popovers#40]
Hi,

I've after running the example app and trying out the standard showroom I've noticed the following:
![Standard arrow](https://user-images.githubusercontent.com/4563901/172191316-6ec3c084-dddd-4589-813f-cb25827491df.png)

The arrow extends over the rounded corner. Looking at the code it is supposed to be fixed by using an "arrowSidePadding" but I guess something isn't working correctly?
I also see this issue when I integrate it into my app.

Version: 1.3.2

**Comment by aheze** [comment]
Indeed, when I was making this template I ran into the exact same problem, hardcoding a padding of `1.8`...

![Screen Shot 2022-06-06 at 9 10 29 PM](https://user-images.githubusercontent.com/49819455/172294238-149bf505-d01e-481d-a2d2-c98bf9984056.png)

Not sure what I might have changed to cause this, I'll investigate. Maybe a device size issue?

**Comment by ConfusedVorlon** [comment]
Just noting that I found exactly the same bug using Popovers on iOS 15.5
Adjusting the arrowSidePadding fixed the issue for me

(in fact - I have tweaked things to make that a configurable option - it's nice to be able to adjust to fit the layout)
